src/sql/platform/telemetry/common/telemetryKeys.ts

```diff
@@ -45,6 +45,7 @@ export const FireWallRule = 'FirewallRule';
 export const AutoOAuth = 'AutoOAuth';
 export const AddNewDashboardTab = 'AddNewDashboardTab';
 export const ProfilerFilter = 'ProfilerFilter';
+export const SelectImage = 'SelectImage';
```

Comment:
the key name is not clear for me what it stands for, select a graphical image? or OS image?

Reply:
@alanrenmsft 
This is used by src/sql/workbench/contrib/notebook/browser/calloutDialog.ts
How about I rename it to: "CalloutDialog_SelectImage" ?

Reply:
Changed to: NotebookCallout